src/usage/context/usage.tsx

```diff
@@ -41,6 +41,13 @@ const DEFAULT_USAGE: Usage = {
   status: RemoteDataState.NotStarted,
 }
 
+export enum VectorName {
+  QueryCount = 'Query Count',
+  Storage = 'Storage',
+  DataIn = 'Data In',
+  DataOut = 'Data Out',
+}
```

Comment:
If we're only using `'Query Count'` today, I think we're better off either inlining the `'Query Count'` string in `/src/usage/UsageSingleStat.tsx` or exporting a `const QueryCount = 'Query Count'` here instead. Otherwise, we're maintaining the other vector names without any confirmation that they're correct, which could lead to someone mistakenly relying on their accuracy in the future.

Reply:
That's a good point -- this was really anticipating future use since I wasn't happy with the fact that we don't identify the actual string values anywhere in the UI. But there is no need for any of the enum values other than `QueryCount` now.